rrule.gemspec

```diff
@@ -23,5 +23,4 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.3.0'
   s.add_runtime_dependency 'activesupport', '>= 4.1'
-  s.add_development_dependency 'rspec', '~> 3.4'
```

Comment:
What's the reason for putting this in the gemfile? You shouldn't have to install rspec just to use the gem, right?

Reply:
Gemfile is ignored for users of the gem. It only matters for developing the gem. `add_development_dependency` vs putting in the Gemfile are functionally equivalent.

Reply:
Got it - sounds good.